src/components/sbb-checkbox-group/sbb-checkbox-group.tsx

```diff
@@ -100,6 +104,30 @@ export class SbbCheckboxGroup implements ComponentInterface {
     this._namedSlots = queryNamedSlotState(this._element, this._namedSlots, event.detail);
   }
 
+  @Listen('keydown')
+  public handleKeyDown(evt: KeyboardEvent): void {
+    const enabledCheckboxes: HTMLSbbCheckboxElement[] = this._checkboxes.filter(
+      (tag: HTMLSbbCheckboxElement) => !tag.hasAttribute('disabled')
+    );
+
+    if (
+      !enabledCheckboxes ||
+      // don't trap nested handling
+      ((evt.target as HTMLElement) !== this._element &&
+        (evt.target as HTMLElement).parentElement !== this._element)
+    ) {
+      return;
+    }
+
+    if (isArrowKeyPressed(evt)) {
+      const current: number = enabledCheckboxes.findIndex(
+        (e: HTMLSbbCheckboxElement) => e === evt.target
+      );
+      const nextIndex: number = getNextElementIndexFunction(evt)(current, enabledCheckboxes.length);
```

Comment:
A matter of taste/readability. Take your pick :)
```suggestion
const getNextElementIndex = getNextElementIndexFunction(evt);
      const current: number = enabledCheckboxes.findIndex(
        (e: HTMLSbbCheckboxElement) => e === evt.target
      );
      const nextIndex: number = getNextElementIndex(current, enabledCheckboxes.length);
```